src/settings/LoanPolicy/components/EditSections/RenewalsSection/RenewalsSection.test.js

```diff
@@ -0,0 +1,327 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+} from '@testing-library/react';
+
+import '../../../../../../test/jest/__mock__';
+
+import { Field } from 'react-final-form';
+
+import {
+  Checkbox,
+  Select,
+  TextField,
+} from '@folio/stripes/components';
+import Period from '../../../../components/Period';
+import {
+  intervalPeriods,
+  renewFromOptions,
+} from '../../../../../constants';
+import RenewalsSection from './RenewalsSection';
+
+jest.mock('../../../../components/Period', () => jest.fn(() => null));
+jest.mock('../../../../utils/options-generator', () => jest.fn((func, options, id) => ({
+  id,
+  options,
+})));
+
+describe('RenewalsSection', () => {
+  const labelIds = {
+    altFDDSDueDateLimit: 'ui-circulation.settings.loanPolicy.altFDDSDueDateLimit',
+    altFDDSforRenewals: 'ui-circulation.settings.loanPolicy.altFDDSforRenewals',
+    renewals: 'ui-circulation.settings.loanPolicy.renewals',
+    renewable: 'ui-circulation.settings.loanPolicy.renewable',
+    unlimitedRenewals: 'ui-circulation.settings.loanPolicy.unlimitedRenewals',
+    numRenewalsAllowed: 'ui-circulation.settings.loanPolicy.numRenewalsAllowed',
+    renewFrom: 'ui-circulation.settings.loanPolicy.renewFrom',
+    pleaseSelect: 'ui-circulation.settings.common.pleaseSelect',
+    renewalPeriodDifferent: 'ui-circulation.settings.loanPolicy.renewalPeriodDifferent',
+    alternateLoanPeriodRenewals: 'ui-circulation.settings.loanPolicy.alternateLoanPeriodRenewals',
+    selectInterval: 'ui-circulation.settings.loanPolicy.selectInterval',
+  };
+  const testIds = {
+    renewals: 'renewalsTestId',
+    renewable: 'renewableTestId',
+    unlimitedRenewals: 'unlimitedRenewalsTestId',
+    numRenewalsAllowed: 'numRenewalsAllowedTestId',
+    renewFrom: 'renewFromTestId',
+    renewalPeriodDifferent: 'renewalPeriodDifferentTestId',
+    alternateLoanPeriodRenewals: 'alternateLoanPeriodRenewalsTestId',
+    altFDDS: 'altFDDSTestId',
+  };
+  const fieldMap = {
```

Comment:
maybe let us call such object something like fieldCallOrderByPlace? Let us discuss

Reply:
it make sense. i'll rename

Reply:
done